stun_server/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  cloud {
+    organization = "home_assistant"
+
+    workspaces {
+      name = "stun_server"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "us_east_1" {
+  source = "./region"
+
+  region      = "us-east-1"
+  domain_name = var.domain_name
+  subdomain   = "stun-us"
```

Comment:
lets concat these with the full region names (can even be handled automatically in the dns.tf file)

Reply:
Do you really think the URLs should include the full region in the subdomain? IMO, it makes them unnecessarily longer and adds unneeded information.

Reply:
Yes, while they are not needed now, we might want more locations in the same continent later

Reply:
All right, if you think there will ever be a need.